src/librustc_mir/build/matches/mod.rs

```diff
@@ -316,21 +323,22 @@ impl<'a,'tcx> Builder<'a,'tcx> {
         // otherwise, extract the next match pair and construct tests
         let match_pair = &candidates.last().unwrap().match_pairs[0];
         let test = self.test(match_pair);
-        debug!("match_candidates: test={:?} match_pair={:?}", test, match_pair);
+        debug!("match_candidates: test={:?} match_pair={:?}",
+               test,
+               match_pair);
         let target_blocks = self.perform_test(block, &match_pair.lvalue, &test);
 
         for (outcome, mut target_block) in target_blocks.into_iter().enumerate() {
-            let applicable_candidates: Vec<Candidate<'tcx>> =
-                candidates.iter()
-                          .filter_map(|candidate| {
-                              unpack!(target_block =
+            let applicable_candidates: Vec<Candidate<'tcx>> = candidates.iter()
+                                                                        .filter_map(|candidate| {
+                                                                            unpack!(target_block =
                                       self.candidate_under_assumption(target_block,
```

Comment:
It's because it's embedded in a non-trivial macro. Pretty sure there is an issue for this already, but couldn't find it.